packages/fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -65,6 +65,44 @@ pub fn expand_function(
     })
 }
 
+/// Generate the function (as a `TokenStream`) used to encode into script data the arguments to the
```

Comment:
Nit: The return value in the signature already conveys the TokenStream part.
```suggestion
Generate the function used to encode arguments to the`main` function of a script.
```